lib/FrontendSortingMCL/FrontendSortingMCL.js

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import orderBy from 'lodash/orderBy';
+
+import {
+  Icon,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import { AcqEndOfList } from '../AcqEndOfList';
+import {
+  ASC_DIRECTION,
+  DESC_DIRECTION,
+} from '../AcqList';
+import { acqRowFormatter } from '../utils';
+
+const alignRowProps = { alignLastColToEnd: true };
+
+const FrontendSortingMCL = ({
+  columnMapping,
+  contentData,
+  formatter,
+  hasArrow,
+  sortDirection,
+  sortedColumn,
+  sorters,
+  visibleColumns,
+  ...rest
+}) => {
+  const resultsFormatter = useMemo(() => ({
+    arrow: () => <Icon icon="caret-right" />,
+    ...formatter,
+  }), [formatter]);
+  const mapping = useMemo(() => ({
+    arrow: null,
```

Comment:
I think Axe identifies this as an issue

Reply:
are there any solutions?

Reply:
Actually, works for ui-invoice, I'll check this later in other modules
![Screenshot from 2020-09-01 03-06-46](https://user-images.githubusercontent.com/43577432/91780451-4f21ed00-ec00-11ea-8c91-4cda5bb9e551.png)